File: Population/population.cpp
#include <bits/stdc++.h>

using namespace std;

double population(int y)
{
    if (y == 2021) return 98.4E+6;
    return population(y - 1) * (1 + 0.01); // the growth rate is 1.00 %
}

int main()
{
    int year;
    cout << "Enter a year: ";
    cin >> year;

    cout << "The population of Vietnam will be " << setprecision(0) << fixed << population(year);
    return 0;
}

File: Population/population.py
def population(year):
    if year == 2021:
        return 98.4E+6

    return population(year - 1) * (1 + 0.01) # The growth rate is 1.00

if __name__ == '__main__':
    year = int(input('Enter a year: '))
    p = population(year)
    print(f'The population of Vietnam will be {p:.0f}')